examples/src/main/java/io/kubernetes/client/examples/PatchExample.java

```diff
@@ -88,6 +90,16 @@ public static void main(String[] args) throws IOException {
                   null);
       System.out.println("strategic-merge-patched deployment" + deploy3);
 
+      // apply-yaml a deployment
```

Comment:
explicitly note that server-side apply is yet alpha feature, and has to be enabled in the server